test/e2e/debug_calltracer_test.go

```diff
@@ -184,7 +189,7 @@ func TestDebugTraceTransactionCallTracer(t *testing.T) {
 
 				// save result in a file
```

Comment:
Can we delete these lines?

Reply:
This code is very helpful to keep investigating new issues with the tracers.

In order to avoid having it duplicated/commented on the tests, I've created a method to allow it to be reused in multiple tests related to trace and also have added a parameter to skip or generate the file.

I think this is a healthy way to clarify and maintain this piece of code.

wdyt?

Reply:
Ok, but we could always go back in time and recover it when needed. But let's keep it.